tests/Integration/Behaviour/Features/Scenario/Carrier/Ranges/carrier_ranges.feature

```diff
@@ -0,0 +1,181 @@
+      | max_width        | 1454                               |
+      | max_height       | 1234                               |
+      | max_depth        | 1111                               |
+      | max_weight       | 3864                               |
+      | group_access     | visitor, guest                     |
+      | delay[en-US]     | Shipping delay                     |
+      | shippingHandling | false                              |
+      | isFree           | true                               |
+      | shippingMethod   | price                              |
+      | taxRuleGroup     | US-AL Rate (4%)                    |
+      | rangeBehavior    | disabled                           |
+    Then I set ranges for carrier "carrier1" called "newCarrier1" with specified properties for all shops:
+      | id_zone | range_from | range_to | range_price |
+      | 1       | 0          | 100      | 10          |
+      | 1       | 100        | 200      | 20          |
+      | 1       | 200        | 300      | 30          |
+      | 1       | 300        | 400      | 40          |
+      | 2       | 0          | 100      | 15          |
+      | 2       | 100        | 200      | 25          |
+    Then carrier "newCarrier1" should have the following ranges for all shops:
+      | id_zone | range_from | range_to | range_price |
+      | 1       | 0          | 100      | 10          |
+      | 1       | 100        | 200      | 20          |
+      | 1       | 200        | 300      | 30          |
+      | 1       | 300        | 400      | 40          |
+      | 2       | 0          | 100      | 15          |
+      | 2       | 100        | 200      | 25          |
+
+  Scenario: Adding weight ranges in carrier
+    When I create carrier "carrier1" with specified properties:
+      | name             | Carrier 1                          |
+      | grade            | 1                                  |
+      | trackingUrl      | http://example.com/track.php?num=@ |
+      | position         | 2                                  |
+      | active           | true                               |
+      | max_width        | 1454                               |
+      | max_height       | 1234                               |
+      | max_depth        | 1111                               |
+      | max_weight       | 3864                               |
+      | group_access     | visitor, guest                     |
+      | delay[en-US]     | Shipping delay                     |
+      | shippingHandling | false                              |
+      | isFree           | true                               |
+      | shippingMethod   | weight                             |
+      | taxRuleGroup     | US-AL Rate (4%)                    |
+      | rangeBehavior    | disabled                           |
+    Then I set ranges for carrier "carrier1" called "newCarrier1" with specified properties for all shops:
```

Comment:
Something comes to mind, should we try a test with the same data (which is valid, no ranges overlap) but not in the same order? Just to be sure the data doesn't need to be ordered in any specific way to work as expected?